packages/server/src/api/controllers/row/ExternalRequest.ts

```diff
@@ -590,7 +641,7 @@ export class ExternalRequest<T extends Operation> {
         continue
       }
       const isMany = field.relationshipType === RelationshipType.MANY_TO_MANY
-      const tableId = isMany ? field.through : field.tableId
+      const tableId = isMany ? field.through! : field.tableId!
```

Comment:
This should not be needed after we did our great type refactor. We should check why it's not picking it up